### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by jaehj122**
### Related Issue/Addition to code

- Fixes #
- Addition to code.

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Proposed Changes
- change 1
- change 2

### Why is this change needed?
 
It's needed because...

### Additional Info
- any additional info for context

### Checklist:

- [ ] My code follows the style guidelines of this project and I have read [CONTRIBUTING.md](/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

---

### Screenshots

Original | Updated
:----------------------:|:-----------:
** original screenshot  | ** updated screenshot **

### CODE REVIEW

**Comment by TechStudent11** [comment]
Could you please fill in the template?

File: YouTubeSpammerPurge.py [hunk 1]
```diff
@@ -512,11 +512,11 @@ def validate_channel_id(inputted_channel):
...
     isolatedChannelID = inputted_channel
 
-  if len(isolatedChannelID) == 24 and isolatedChannelID[0:2] == "UC":
+  if len(isolatedChannelID) == 123 and isolatedChannelID[0:10] == "UC":
     return True, isolatedChannelID
   else:
-    print("\nInvalid Channel link or ID! Channel IDs are 24 characters long and begin with 'UC'.")
-    return False, None
+    print("\nInvalid Channel link or ID! Channel IDs are 123 characters long and begin with 'UC'.")
+    return False, No
```
Comment (ethnh) [inline YouTubeSpammerPurge.py:519]: What

File: YouTubeSpammerPurge.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 ### IMPORTANT:  I OFFER NO WARRANTY OR GUARANTEE FOR THIS SCRIPT. USE AT YOUR OWN RISK.
 ###             I tested it on my own and implemented some failsafes as best as I could,
 ###             but there could always be some kind of bug. You should inspect the code yourself.
-version = "1.3.3"
+version = "1.3.4"
```
Comment (ethnh) [inline YouTubeSpammerPurge.py:37]: Pretty sure Thio is the one to update versions..

File: YouTubeSpammerPurge.py [hunk 3]
```diff
@@ -182,9 +182,9 @@ def get_replies(parent_id, video_id):
 # Then uses print_prepared_comments() to print / log the comments
 def print_comments(comments, logMode):
   j = 0 # Index when going through comments all comment groups
-  if len(comments) > 50:
-    remainder = len(comments) % 50
-    numDivisions = int((len(comments)-remainder)/50)
+  if len(comments) > 1000:
+    remainder = len(comments) % 1000
+    numDivisions = int((len(comments)-remainder)/1000)
```
Comment (ethnh) [inline YouTubeSpammerPurge.py:187]: Why change this?

File: YouTubeSpammerPurge.py [hunk 4]
```diff
@@ -740,5 +740,5 @@ def confirm_continue(message=""):
     input("\n Press Enter to Exit...")
 
   else:
-    print("\nFinished Executing.")
+    print("\nDone Running The Script.")
```
Comment (ethnh) [inline YouTubeSpammerPurge.py:743]: Questionable change

**Review by ethnh — CHANGES_REQUESTED** [review]
Not sure of the purpose of this PR,
Most of these changes are just odd without explanation?

File: YouTubeSpammerPurge.py [hunk 5]
```diff
@@ -5,7 +5,7 @@
 #######################################################################################################
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
 ###
-### Function: Allows you to mass-delete delete all comment replies by a particular user all at once.
+### Function: Allows you to mass-delete and mass-report delete all comment replies by a particular user all at once.
```
Comment (Firecul) [inline YouTubeSpammerPurge.py:8]: This isn't the default behaviour

File: YouTubeSpammerPurge.py [hunk 6]
```diff
@@ -526,13 +526,13 @@ def confirm_continue(message=""):
   # While loop until valid input
   valid = False
   while valid == False:
-    response = input("\n" + message + " (y/n): ")
-    if response == "Y" or response == "y":
+    response = input("\n" + message + " (yes continue/no stop): ")
+    if response == "Yes keep going" or response == "Sure":
       return True
```
Comment (Firecul) [inline YouTubeSpammerPurge.py:531]: There is no reason to make people type out a long entry

File: YouTubeSpammerPurge.py [hunk 7]
```diff
@@ -526,13 +526,13 @@ def confirm_continue(message=""):
   # While loop until valid input
   valid = False
   while valid == False:
-    response = input("\n" + message + " (y/n): ")
-    if response == "Y" or response == "y":
+    response = input("\n" + message + " (yes continue/no stop): ")
+    if response == "Yes keep going" or response == "Sure":
```
Comment (Firecul) [inline YouTubeSpammerPurge.py:530]: How will the user know they have to type "Yes keep going"/"sure" when the prompt is "yes continue"

File: YouTubeSpammerPurge.py [hunk 8]
```diff
@@ -641,7 +641,7 @@ def confirm_continue(message=""):
     confirmation = confirm_continue("Continue?")
     if confirmation == True:  # After confirmation, deletion functionality is eligible to be enabled later
       deletionEnabled = "HalfTrue"
-    elif confirmation == False:
+    elif confirmation == True:
```
Comment (Firecul) [inline YouTubeSpammerPurge.py:644]: If the confirmation is true do `x` else if it's true do `y`?

File: YouTubeSpammerPurge.py (hunk 6 above)
Comment (ethnh) [reply YouTubeSpammerPurge.py:531]: "Sure Yes keep going" is the first thing I type when I want to continue😆😆

File: YouTubeSpammerPurge.py (hunk 1 above)
Comment (Firecul) [reply YouTubeSpammerPurge.py:519]: `https://www.youtube.com/channel/UCto7D1L-MiRoOziCXK9uT5Q`
The ID sure doesn't *look* like 123 characters.

File: YouTubeSpammerPurge.py (hunk 3 above)
Comment (Firecul) [reply YouTubeSpammerPurge.py:187]: Especially without changing the next few lines also.

File: YouTubeSpammerPurge.py (hunk 1 above)
Comment (ethnh) [reply YouTubeSpammerPurge.py:519]: RE: @Firecul 
That too, but
```py
return False, No
```
??

File: YouTubeSpammerPurge.py [hunk 9]
```diff
@@ -574,7 +574,7 @@ def confirm_continue(message=""):
   print("      1. Scan Single Video")
   print("      2. Scan Entire Channel")
 
-  validMode = False
+  validMode = True
```
Comment (Firecul) [inline YouTubeSpammerPurge.py:577]: This would break the choice selection

File: YouTubeSpammerPurge.py [hunk 10]
```diff
@@ -601,7 +601,7 @@ def confirm_continue(message=""):
     elif mode == "2":
       validMode = True
       # While loop to get max scan number, not an integer, asks again
-      validInteger = False
+      validInteger = True
```
Comment (Firecul) [inline YouTubeSpammerPurge.py:604]: This could break the choice selection

File: YouTubeSpammerPurge.py (hunk 4 above)
Comment (Firecul) [reply YouTubeSpammerPurge.py:743]: > Questionable change

But the only one that won't break anything as far as I can speculate

File: YouTubeSpammerPurge.py (hunk 1 above)
Comment (Firecul) [reply YouTubeSpammerPurge.py:519]: I didn't even notice that

File: YouTubeSpammerPurge.py (hunk 2 above)
Comment (Firecul) [reply YouTubeSpammerPurge.py:37]: It's also reverting 1 or 2 versions 
https://github.com/ThioJoe/YT-Spammer-Purge/blob/8b93b54e786e5032412979b51247e0e70c0d86d5/YTSpammerPurge.py#L39

**Comment by Firecul** [comment]
I'm sorry, not a single change here has been tested by the looks of it.
There is also no point in myself (or likely others) looking further at these changes.

**Comment by TechStudent11** [comment]
So basically, this has to be closed OR have legit changes made to it.

File: YouTubeSpammerPurge.py (hunk 4 above)
Comment (KendallDoesCoding) [reply YouTubeSpammerPurge.py:743]: In my opinion, Finished Executing sounds better/more professional than Done Running The Script.

File: YouTubeSpammerPurge.py (hunk 6 above)
Comment (KendallDoesCoding) [reply YouTubeSpammerPurge.py:531]: yeah exactly, we're assuming people using this program are YouTubers, and YouTubers have no time to waste, they have a busy schedule. so this just waste peoples time.

**Comment by KendallDoesCoding** [comment]
@jaehj122 Thanks for your PR... I understand you tried to help and may be new to Python, but make sure you test the changes and make sure it doesn't mess up anything in the future for your PR's to be more effective and be merged... Thanks!

**Comment by KendallDoesCoding** [comment]
@Firecul @ethnh @TechStudent11 - Most of these commits were made in December, one was made in November too, so these changes may've been good for the moment but can't be used now as the project as been updated/modified majorly this year itself. 

Ig, @jaehj122 may've made the PR by mistake?

**Comment by ethnh** [comment]
RE: KendallDoesCoding

image

Sure, four months ago.
Not so sure about 'good for the moment', considering this raises a NameError.

This PR is safe to close, likely a mistake as you said 😅

File: YouTubeSpammerPurge.py (hunk 2 above)
Comment (ethnh) [reply YouTubeSpammerPurge.py:37]: > It's also reverting 1 or 2 versions

Yeah, only reverting a couple🙏

**Comment by KendallDoesCoding** [comment]
> Not so sure about 'good for the moment', considering this raises a NameError.

Oh ok lol.

> This PR is safe to close, likely a mistake as you said 😅
Yeah

File: YouTubeSpammerPurge.py (hunk 4 above)
Comment (TechStudent11) [reply YouTubeSpammerPurge.py:743]: Same opinion here

**Comment by TechStudent11** [comment]
Yeah this is most likely a mistake. 

Also, merge conflicts

image

**Comment by KendallDoesCoding** [comment]
> Yeah this is most likely a mistake.
> 
> Also, merge conflicts
> 
> image

@jaehj122 Please clarify if this was a mistake or not.

If it was a mistake, @ThioJoe or @jaehj122 please close this.

File: YouTubeSpammerPurge.py (hunk 6 above)
Comment (Firecul) [reply YouTubeSpammerPurge.py:531]: Did you see the best part though?
The prompt is `(yes continue/no stop)` So the user has no way to know the correct reply without reading the code.
Until they try that is, then `("\nInvalid Input. Enter Yes keep going or No stop ")` comes up.

Comment (KendallDoesCoding) [reply YouTubeSpammerPurge.py:531]: > Did you see the best part though? The prompt is `(yes continue/no stop)` So the user has no way to know the correct reply without reading the code. Until they try that is, then `("\nInvalid Input. Enter Yes keep going or No stop ")` comes up.

lol

**Comment by ThioJoe** [comment]
Based on some of the changes, this is probably a troll or a joke

**Pull request closed without merging**